### PR TITLE
Misc tweaks that should survive after the MAK exhibition

### PR DESCRIPTION
**Pull request by janper**
See inline comments for more details about the individual tweaks

### CODE REVIEW

File: src/interpreter/ast.rs
Comment (janper) [inline src/interpreter/ast.rs:135]: No more dead (or at least Clippy doesn't complain)

File: src/interpreter/ast.rs
Comment (janper) [inline src/interpreter/ast.rs:161]: No more dead (or at least Clippy doesn't complain)

File: src/interpreter/ast.rs
Comment (janper) [inline src/interpreter/ast.rs:163]: No more dead (or at least Clippy doesn't complain)

File: src/interpreter_funcs/voxelize.rs
Comment (janper) [inline src/interpreter_funcs/voxelize.rs:15]: We can afford more on the NUC, which is our reference hardware. Actually, we can afford even more than this (10M voxel operations take about 15s).

File: src/interpreter_funcs/voxelize.rs
Comment (janper) [inline src/interpreter_funcs/voxelize.rs:21]: Forgotten erroneous scenario.

File: src/interpreter_funcs/voxelize.rs
Comment (janper) [inline src/interpreter_funcs/voxelize.rs:89]: Small values, even though they may not generate too many voxels, cause the calculation to be too small. It's because the `to_mesh` function needs to divide each face into chunks as small as the smallest voxel dimension, which makes the voxelization too slow. `0.005` is an arbitrary number but seems to work well. The minimum value is being set very often because the user drags the slider all the was to the left quite often.

File: src/interpreter_funcs/voxelize.rs
Comment (janper) [inline src/interpreter_funcs/voxelize.rs:163]: The rest of the code is slightly nicer when the voxel dimensions are already stored in a Vector

File: src/interpreter_funcs/voxelize.rs
Comment (janper) [inline src/interpreter_funcs/voxelize.rs:178]: Check if the vector dimensions are above zero. Should never happen thanks to the GUI settings. Maybe can be turned into an assert and panic if it happens. I didn't want to risk it at the exhibition though.

File: src/interpreter_funcs/voxelize.rs
Comment (janper) [inline src/interpreter_funcs/voxelize.rs:188]: This is the nicer part. 
However, this is a very temporary piece of code because it's being replaced by a more precise method in PR#154

File: src/lib.rs
Comment (janper) [inline src/lib.rs:185]: Start with a maximized window

File: src/lib.rs
Comment (janper) [inline src/lib.rs:208]: We don't know the windows size upfront because fullscreen can be anything.

File: src/ui.rs
Comment (janper) [inline src/ui.rs:40]: Make this a global constant so that we can change it easily at one place.

File: src/ui.rs
Comment (janper) [inline src/ui.rs:503]: This makes the about window fixed height and therefore it doesn't extend the window boundaries (so easily).

File: src/ui.rs
Comment (janper) [inline src/ui.rs]: The description has changed. This is what we exhibit.

File: src/ui.rs
Comment (janper) [inline src/ui.rs]: Fixes the orange background of the dropdown menus.

File: src/interpreter_funcs/voxelize.rs
Comment (yanchith) [reply src/interpreter_funcs/voxelize.rs:178]: Interesting thought, but let's pretend the UI can't validate anything. This will make us future-proof for other frontends.

File: src/lib.rs
Comment (yanchith) [reply src/lib.rs:185]: We'll need to test this on macOS before merging. If this works well, I think we can get rid of the fullscreen (`HS_FULLSCREEN=1`) option completely. The removal can be a separate PR.

File: src/ui.rs
Comment (yanchith) [inline src/ui.rs]: formatting

Comment (yanchith) [reply src/ui.rs]: Should we also update the initial README section to this text?

**Review by yanchith — COMMENTED** [review]
Nice! Almost no objections, see inline comments for questions.

File: src/interpreter_funcs/voxelize.rs
Comment (janper) [reply src/interpreter_funcs/voxelize.rs:178]: That's a relief! Better safe than sorry!

File: src/lib.rs
Comment (janper) [reply src/lib.rs:185]: I'd like to keep the fullscreen option too. May be useful for some showcases.

File: src/ui.rs
Comment (janper) [reply src/ui.rs]: Not yet. We'll come back to it before the end of the project. 
We'll have to mention the sponsors too... Let's postpone.

File: src/lib.rs
Comment (yanchith) [reply src/lib.rs:185]: Just tested on linux (X11) :heavy_check_mark:

**Review by yanchith — APPROVED** [review]
I'll merge this in the evening once I test on macOS.

**Comment by janper** [comment]
Please, merge #151 and #152 first. There may be conflicts.

**Pull request merged**
